docs/Reference/Console/overview.md

```diff
@@ -0,0 +1,26 @@
+--8<-- "References/abbreviations.md"
+
+!!! warning "Warning - Console in Alpha Development"
+    Console is currently under development. The current available tags for download are Alpha version code. This could mean that certain features may not function yet, visual look and feel may change, or bugs/errors may occur. It is not recommended for use in Production deployments. Follow along our [Feature Backlog for future releases and feature updates](https://github.com/orgs/open-amt-cloud-toolkit/projects/10)
+
+Connect to devices in a 1:1, direct connection over a shared local network.
+
+Console is an application that packages the Sample Web UI, RPS, and MPS into a single executable for use in an enterprise environment. Users can add activated AMT devices to access device information and device management functionality such as power control, remote keyboard-video-mouse (KVM) control, and more.
```

Comment:
While this statement is conceptually accurate, it isn't technically accurate.  If we are going to state it this way, we should be clear that we re-implemented RPS and MPS into a single Go application to provide similar use cases for the enterprise environment (or something along those lines).